lib/jsonapi/utils/exceptions.rb

```diff
@@ -4,20 +4,57 @@ module JSONAPI
   module Utils
     module Exceptions
       class ActiveRecord < ::JSONAPI::Exceptions::Error
-        attr_accessor :object
+        attr_accessor :object, :resource, :relationships, :relationship_keys, :foreign_keys
 
-        def initialize(object)
+        def initialize(object, request, context)
```

Comment:
I think that would be a good change. I think I was passing in the entire request because I wasn't sure what all I would need from it while poking around.